tasks/minimize_access.yml

```diff
@@ -48,3 +48,27 @@
     group: 'root'
     mode: '0750'
   when: '"change_user" not in os_security_users_allow'
+
+- name: Ensure file /etc/fstab exists
+  stat:
+    path: /etc/fstab
+  register: fstab_file
+
+- name: Create proc mount point
```

Comment:
Is this task necessary? Are the distributions that don't have this directory?

Reply:
Same as above.